client/blocks/feedback/edit.js

```diff
@@ -248,12 +249,15 @@ const EditFeedbackBlock = ( props ) => {
 			null !== attributes.closedAfterDateTime &&
 			new Date().toISOString() > attributes.closedAfterDateTime );
 
-	const hideBranding = get( accountInfo, 'capabilities' ).includes(
+	const hideBranding = get( accountInfo, 'capabilities', [] ).includes(
 		'hide-branding'
 	);
 
 	return (
-		<ConnectToCrowdsignal>
+		<ConnectToCrowdsignal
+			blockName={ __( 'Feedback Button', 'crowdsignal-forms' ) }
+			blockIcon=<FeedbackIcon />
```

Comment:
Missing braces.

Reply:
Wrapping `<FeedbackIcon />`? Weird it didn't blow up, but I'll them if that's the case.